macros/directory_pattern.sql

```diff
@@ -0,0 +1,24 @@
+-- these macros will read a user’s home environment and detect whether a computer’s operating system is Windows based or Mac/Linux, and display the right directory pattern.
+{% macro directory_pattern() %}
```

Comment:
Should we add adapter.dispatch to these macros since they're nested within a package?